XLAL Error - XLALFrameUFrChanRead_FrameL_ (LALFrameL.c:882): Channel H1:LDAS-STRAIN could not be read: error code FR_ERROR_CHECKSUM
XLAL Error - XLALFrameUFrChanRead_FrameL_ (LALFrameL.c:882): I/O error
XLAL Error - XLALFrFileReadREAL8TimeSeries_ (LALFrameIO_source.c:61): Wrong name
XLAL Error - XLALFrStreamGetREAL8TimeSeries (LALFrStreamReadTS_source.c:132): Internal function call failed: Wrong name
XLAL Error - XLALFrStreamReadREAL8TimeSeries (LALFrStreamReadTS_source.c:236): Internal function call failed: Wrong name
Traceback (most recent call last):
  File "/home/spxha/solarGW/solarbayes/likelihood_job.py", line 49, in <module>
    strainH = TimeSeries.read(pathtoinput+'S6framesH1.lcf',channel='H1:LDAS-STRAIN', start=starttime, end=endtime)
  File "/home/spxha/lib/python2.7/site-packages/astropy-1.0.6-py2.7-linux-x86_64.egg/astropy/io/registry.py", line 332, in read
    data = reader(*args, **kwargs)
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/timeseries/io/cache.py", line 172, in read_cache
    resample=resample, **kwargs)
  File "/home/spxha/lib/python2.7/site-packages/astropy-1.0.6-py2.7-linux-x86_64.egg/astropy/io/registry.py", line 332, in read
    data = reader(*args, **kwargs)
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/utils/deps.py", line 82, in wrapper
    return func(*args, **kwargs)
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/timeseries/io/gwf/__init__.py", line 131, in read_timeseries
    return read_timeseriesdict(source, [channel], *args, **kwargs)[channel]
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/timeseries/io/gwf/__init__.py", line 125, in read_timeseriesdict
    return reader(source, *args, **kwargs)
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/utils/deps.py", line 82, in wrapper
    return func(*args, **kwargs)
  File "/home/spxha/.local/lib/python2.7/site-packages/gwpy-0.1b3.dev229-py2.7.egg/gwpy/timeseries/io/gwf/lalframe.py", line 114, in read_timeseriesdict
    verbose=verbose)
  File "/home/spxha/lib/python2.7/site-packages/lalframe/frread.py", line 144, in read_timeseries
    verbose=verbose)
  File "/home/spxha/lib/python2.7/site-packages/lalframe/frread.py", line 180, in ts_from_cache
    datatype=datatype, verbose=verbose)
  File "/home/spxha/lib/python2.7/site-packages/lalframe/frread.py", line 247, in ts_from_stream
    datatype=datatype))
  File "/home/spxha/lib/python2.7/site-packages/lalframe/frread.py", line 265, in read_channel_from_stream
    origin = read(stream, channel, start, duration, 0)
RuntimeError: Internal function call failed: Wrong name
